src/Sarif.WorkItems/SarifWorkItemsExtensions.cs

```diff
@@ -85,5 +116,28 @@ private static string ConstructFullRuleIdentifier(ReportingDescriptor reportingD
             }
             return fullRuleIdentifier;
         }
+
+        public static string CreateWorkItemDescription(this SarifLog log)
+        {
+            Dictionary<string, int> resultCountsByTool = FetchToolResultCounts(log);
+            StringBuilder templateText = new StringBuilder(@"This bug has been filed by the Sarif Work Item Automatic Filer.  It contains results for the following tools and issues:");
```

Comment:
>This bug has been filed by the Sarif Work Item Automatic Filer.  It contains results for the following tools and issues: [](start = 61, length = 120)

Larry and I can work with you offline on the text. I can tell you that I don't want to burn the  name of our filer in this message. The focus is not on us, it is on the bug content. :) 

Just a note. We need to think about how the system will dynamically produce some data while allow other pieces to be configurable. We can define things on the command-line and in context for things like a support alias.

Reply:
"This bug was filed automatically. ..."

---
In reply to: [394698234](https://github.com/microsoft/sarif-sdk/pull/1817#discussion_r394698234) [](ancestors = 394698234)